pkg/Microsoft.NETCore.Platforms/runtime.json

```diff
@@ -922,18 +915,11 @@
             "#import": [ "fedora-corert", "linux-x64-corert", "fedora-x64" ]
         },
 
-        "fedora.23-corert": {
-            "#import": [ "fedora-corert", "fedora.23" ]
-        },
-        "fedora.23-x64-corert": {
-            "#import": [ "fedora.23-corert", "fedora-x64-corert", "fedora.23-x64" ]
-        },
-
         "fedora.24-corert": {
-            "#import": [ "fedora.23-corert", "fedora.24" ]
+            "#import": [ "fedora.24" ]
         },
         "fedora.24-x64-corert": {
-            "#import": [ "fedora.24-corert", "fedora.23-x64-corert", "fedora.24-x64" ]
```

Comment:
IMO, fedora 24 depending on fedora 23 was kind of broken, or at least didn't match anything how deps are set up elsewhere in the RID graph. Thanks for fixing this!